src/pykiso/logging_initializer.py

```diff
@@ -114,8 +116,10 @@ def filter(self, record):
     # if log_path is given create a file handler
     if log_path is not None:
         log_path = Path(log_path)
-        if log_path.is_dir():
-            fname = time.strftime("%Y-%m-%d_%H-%M-test.log")
+        if log_path.suffix == "":
+            if not log_path.is_dir():
```

Comment:
Maybe more 
```
if log_path.is_dir():
    log_path.mkdir(exist_ok=True, parent=True)
``` 
?

Reply:
Well if I do this is_dir will return False if the folder does not exist and it will not be created

Reply:
but previous check is indeed useless, removed

Reply:
Absolutely right!
You could still flatten this a bit by doing:
```
if log_path.suffix == "":
    log_path.mkdir(parents=True, exist_ok=True)
```
which would already take care of the additional is_dir condition

Reply:
Done